cmd/initContainer/main.go

```diff
@@ -73,6 +72,7 @@ func main() {
 	// os signal handler
 	stopCh := signal.SetupSignalHandler()
 	// create client config
+	kubeconfig = ""
```

Comment:
We should clean this up... I will do in in a follow up PR.

Reply:
ok...will you do it? or should I remove it?

Reply:
What do you prefer ?

Reply:
i'll remove it.....ig there is nothing to add here?

Reply:
done in this only......:)

Reply:
You will need to change `config.CreateClientConfig(kubeconfig, clientRateLimitQPS, clientRateLimitBurst)`.
Please, remove it from everywhere.

Reply:
Nope, just the `kubeconfig` argument.

Reply:
this will give an error saying less arguments

Reply:
can you do the required changes? .....  :)

Reply:
Yup, let's merge this one and I will take care of the rest ;-)